File: compiled_lang/cs/src/g20.array.cs
using System;

class G20Array
{
    static void Main()
    {
        string[] nicknames = { "bob", "ed", "steve", "ralph", "joe", "deb", "kate" };

        Console.WriteLine("The names are: ");
        for (int i = 0; i < nicknames.Length; i++)
        {
            Console.WriteLine(" nicknames[" + i + "]=" + nicknames[i]);
        }
    }
}

File: compiled_lang/rust/src/g20.array.rs
fn main() {
    let nicknames = ["bob", "ed", "steve", "ralph", "joe", "deb", "kate"];

    println!("The names are: ");
    for (i, name) in nicknames.iter().enumerate() {
        println!(" nicknames[{i}]={name}");
    }
}
